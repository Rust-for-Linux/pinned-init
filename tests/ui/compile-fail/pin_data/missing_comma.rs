use pin_init::*;

#[pin_data]
struct Foo {
    a: Box<Foo>
    b: Box<Foo>
}
