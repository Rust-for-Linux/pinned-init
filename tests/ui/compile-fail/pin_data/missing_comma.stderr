error: expected `,`, or `}`, found `b`
 --> tests/ui/compile-fail/pin_data/missing_comma.rs:5:16
  |
5 |     a: Box<Foo>
  |                ^ help: try adding a comma: `,`

error: expected `,`
 --> tests/ui/compile-fail/pin_data/missing_comma.rs:6:5
  |
6 |     b: Box<Foo>
  |     ^
